Game2048Player/src/main/java/playgame/PlayGame.java

```diff
@@ -1,87 +1,10 @@
 package playgame;
 
-import javalib.funworld.World;
-import javalib.funworld.WorldScene;
-import javalib.worldimages.TextImage;
-import javalib.worldimages.WorldEnd;
-import models.game2048.Game2048;
-import models.game2048.KeyEventHandler;
-import models.game2048.Scoreboard;
-import models.grid2048.Grid2048;
-import models.square.Square;
-import java.awt.Color;
+public interface PlayGame {
```

Comment:
I like the idea of splitting out some of the shared constants, but this class name no longer makes sense. GamePlayConstants or something?